Cargo.toml

```diff
@@ -39,6 +39,9 @@ members = [
     "tools/restaked"
 ]
 
+[dependencies]
+actix-rt = "=1.1.1"
```

Comment:
Let's have a comment right there with the link to the issue and some context, so it is immediately obvious why this was done. Do you think it makes sense to pin it instead of just requiring `1.1.1`, which in semver means 1.1.1+?

Reply:
I think it makes sense to pin the exact version for now. This is only intended to serve as a temporary solution. A longer term solution would be to update all the actix libraries when they update `actix-rt` version.